SpywareFilter/sections/specific.txt

```diff
@@ -52,6 +52,8 @@
 !
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de
 !
+||dnh523js9661q.cloudfront.net/apis/apTracker.
```

Comment:
Blocked this as a tracker server, please remove it from the pull request.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/116581/commits/7be377c9c2e7d205e46bd065c9f48a8db2fd4d2b
Done.